### PR TITLE
Introduce `lb` AWS module

### PR DESCRIPTION
**Pull request by gmargaritis**
Resolves https://github.com/withlogicco/infrastructure/issues/67

### CODE REVIEW

**Review by akalipetis — COMMENTED** [review]
Just two comments, in general LGTM.

File: aws/lb/main.tf [hunk 1]
```diff
@@ -0,0 +1,139 @@
+locals {
+  tags = {
+    Environment = var.environment
+    Project     = var.project
+    Domain      = var.domain
+  }
+}
+
+data "aws_elb_service_account" "default" {}
```
[the remark this answers is not in the document]
Comment (parisk) [reply aws/lb/main.tf:9]: No, it's static.

**Review by parisk — APPROVED** [review]
LGTM 🚀

File: aws/lb/main.tf [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+resource "aws_s3_bucket" "lb_logs" {
+  bucket = "${var.project}-${var.environment}-lb-logs"
+  tags   = local.tags
+}
```
[the remark this answers is not in the document]
Comment (parisk) [reply aws/lb/main.tf:14]: Let's leave it as is for now to keep with the naming convention, without extra complexity, until we need it.

**Pull request merged**